xbmc/platform/win32/WIN32Util.cpp

```diff
@@ -1228,3 +1228,152 @@ bool CWIN32Util::SetThreadLocalLocale(bool enable /* = true */)
   return _configthreadlocale(param) != -1;
 }
 
+void CWIN32Util::ToggleWindowsHDR()
```

Comment:
Leave this where it is, but rename to setHDR()

Then override the setHDR method in windowing/windows/winsystem32.cpp and just call this. Specifics can be seen in another comment with an example of android doing the same thing.

Reply:
Again:  this is for other purpose than setHDR()